frontend/src/views/Members.vue

```diff
@@ -413,36 +387,73 @@ export default {
         this.kubeconfigDialog = true
       }
     },
-    confirmDelete (username) {
-      const memberName = escape(displayName(username))
+    async onRemoveUser (name) {
+      const removalConfirmed = await this.confirmRemoveUser(name)
+      if (!removalConfirmed) {
+        return
+      }
+      await this.deleteMember(name)
+      if (this.isCurrentUser(name) && !this.isAdmin) {
+        if (this.projectList.length > 0) {
+          const p1 = this.projectList[0]
+          this.$router.push({ name: 'ShootList', params: { namespace: p1.metadata.namespace } })
+        } else {
+          this.$router.push({ name: 'Home', params: { } })
+        }
+      }
+    },
+    confirmRemoveUser (name) {
+      const memberName = escape(displayName(name))
       const projectName = escape(this.projectDetails.projectName)
       let messageHtml
-      if (this.isCurrentUser(username)) {
+      if (this.isCurrentUser(name)) {
         messageHtml = `Do you want to remove <span class="red--text text--darken-2 font-weight-bold">yourself</span> from the project <i>${projectName}</i>?`
       } else {
-        messageHtml = `Do you want to delete the member <i>${memberName}</i> from the project <i>${projectName}</i>?`
+        messageHtml = `Do you want to remove the member <i>${memberName}</i> from the project <i>${projectName}</i>?`
       }
       return this.$refs.confirmDialog.waitForConfirmation({
-        confirmButtonText: 'Delete',
-        captionText: 'Confirm Member Deletion',
+        confirmButtonText: 'Remove Member',
+        captionText: 'Confirm Remove Member',
         messageHtml,
         dialogColor: 'red'
       })
     },
-    async onDelete (username) {
-      const deletionConfirmed = await this.confirmDelete(username)
+    async onDeleteServiceAccount (serviceAccountName) {
+      let deletionConfirmed
+      if (isForeignServiceAccount(this.namespace, serviceAccountName)) {
+        deletionConfirmed = await this.confirmRemoveForeignServiceAccount(serviceAccountName)
+      } else {
+        deletionConfirmed = await this.confirmDeleteServiceAccount(serviceAccountName)
+      }
       if (deletionConfirmed) {
-        await this.deleteMember(username)
-        if (this.isCurrentUser(username) && !this.isAdmin) {
-          if (this.projectList.length > 0) {
-            const p1 = this.projectList[0]
-            this.$router.push({ name: 'ShootList', params: { namespace: p1.metadata.namespace } })
-          } else {
-            this.$router.push({ name: 'Home', params: { } })
-          }
-        }
+        await this.deleteMember(serviceAccountName)
```

Comment:
```suggestion
        return this.deleteMember(serviceAccountName)
```
and removing async from the function should also work, right?

Reply:
As there are no further statements in the method, I guess we can remove the await here